go.mod

```diff
@@ -58,10 +58,14 @@ require (
 	gotest.tools/v3 v3.4.0
 )
 
+require (
+	github.com/Microsoft/hcsshim v0.10.0-rc.7
+	github.com/moby/sys/symlink v0.2.0 // indirect
+)
```

Comment:
Direct dependencies should be in the first `require()` section, and indirect ones should be in the second `require()` section.